### ISSUE TITLE
Deferred whitespace and formatting for PR #1695

### ISSUE THREAD

**Issue by artntek** [NCEAS/metacat#1701]
Some of the files in PR #1695 need significant whitespace and formatting changes, but these files are also being touched in Jing's ["Remove Metacat API" branch](https://github.com/NCEAS/metacat/tree/feature-1365-remove-metacat-api-tables). We're therefore waiting on doing the whitespace and formatting changes until after both branches are merged back to develop, in order to avoid merge conflict hell. The following files need to be reformatted and stripped of tabs:

* `src/edu/ucsb/nceas/metacat/DocumentImpl.java`
* `src/edu/ucsb/nceas/metacat/MetacatHandler.java`
* `src/edu/ucsb/nceas/metacat/dataone/CNodeService.java`

**Comment by artntek** [comment]
addressed in https://github.com/NCEAS/metacat/pull/1725
